vc_assistance/indico_vc_assistance/templates/vc_assistance_request_link.html

```diff
@@ -0,0 +1,15 @@
+{% from 'message_box.html' import message_box %}
+{% if has_vc_room_attached and not request_accepted %}
+    {% call message_box('highlight') %}
+        {% set link_start -%}
+            <a href="{{ url_for('requests.event_requests_details', req) }}">
+        {%- endset %}
+        {% set link_end -%}
+            </a>
+        {%- endset %}
+        {% trans -%}
+            Videoconference rooms have been added with no assistance requested,
```

Comment:
not sure about the wording. this sounds like the user *should* request assistance. what about something like this?

> If you are not familiar with the video conference equipment in the room, you can [request assistance].